proc/proc_darwin.go

```diff
@@ -81,11 +82,48 @@ func Launch(cmd []string, wd string) (*Process, error) {
 		C.free(unsafe.Pointer(argvSlice[i]))
 	}
 
+	for {
```

Comment:
nit: Could you add some more comments to these new lines, just explaining what's happening during intitialization. It's a tricky process, so we should be describing it well.